libraries/matchers.rb

```diff
@@ -0,0 +1,10 @@
+if defined?(ChefSpec)
```

Comment:
Nope, following [chefspec documentation](https://github.com/sethvargo/chefspec#user-content-packaging-custom-matchers) matchers should be in the cookbook libraries. It's also way more convenient to use because chefspec will automatically load your cookbooks libraries but not your cookbooks spec libraries :)